Source/DedicatedServer/Public/ServerConsole.h

```diff
@@ -6,19 +6,20 @@ DECLARE_LOG_CATEGORY_EXTERN( LogServerConsole, Log, All );
 
 #if WITH_SERVER_CODE
 	#if PLATFORM_WINDOWS
-		#include "Private/Windows/WindowsPlatformOutputDevicesPrivate.h"
-		#define FOutputDeviceConsolePlatform FOutputDeviceConsoleWindows
+		#include "Public/Windows/WindowsConsoleOutputDevice.h"
+		#include "Public/Windows/WindowsHWrapper.h"
+		#define FOutputDeviceConsolePlatform FWindowsConsoleOutputDevice
 	#elif PLATFORM_MAC
-		#include "Private/Mac/MacPlatformOutputDevicesPrivate.h"
-		#define FOutputDeviceConsolePlatform FOutputDeviceConsoleMac
+		#include "Public/Mac/MacConsoleOutputDevice.h"
+		#define FOutputDeviceConsolePlatform FLinuxConsoleOutputDevice
 	#elif PLATFORM_LINUX
-		#include "Private/Linux/LinuxPlatformOutputDevicesPrivate.h"
-		#define FOutputDeviceConsolePlatform FOutputDeviceConsoleLinux
+		#include "Public/Linux/LinuxConsoleOutputDevice.h"
+		#define FOutputDeviceConsolePlatform FLinuxConsoleOutputDevice
 	#else
 		#error How did you get here?
 	#endif
 
-	class FServerConsole : public FOutputDeviceConsolePlatform
+	class FServerConsole : public FOutputDeviceConsole
```

Comment:
If you remove `FOutputDeviceConsolePlatform` here then you might as well remove it generally, at a glance it doesn't look like I've used anything that isn't present in `FOutputDevice`